ci/common/test-pyclaragenomics.sh

```diff
@@ -34,17 +34,26 @@ python setup_pyclaragenomics.py --build_output_folder cga_build
 
 logger "Run Tests..."
 run_tests
-
-cd $PYCLARAGENOMICS_DIR
+cd "$PYCLARAGENOMICS_DIR"
 
 logger "Uninstall pyclaragenomics..."
 pip uninstall -y pyclaragenomics
 
 logger "Create pyclaragenomics Wheel package..."
-python setup_pyclaragenomics.py --build_output_folder cga_build_wheel --create_wheel_only
+CUDA_VERSION=$(cat "$CUDA_HOME"/version.txt | cut -d" " -f3 | cut -d"." -f1-2 | sed -e "s/\./_/g")
```

Comment:
is this not available in the jenkins environment? I remember seeing this variable in the verbose jenkins output

Reply:
You're right, thanks!